bundles/GraphicServiceOrder/Service/OrderService.php

```diff
@@ -282,7 +282,7 @@ private function createFolder($order_key)
     }
 
     /**
-     * Share file in owncloud.
+     * Share file in Owncloud.
```

Comment:
It's actually called “ownCloud” (cf. https://owncloud.org/trademark/).

Reply:
Fixed